### PR TITLE
 [deps] Update to openwisp-utils 0.7

### PR DESCRIPTION
**Pull request by atb00ker**
Blocked by https://github.com/openwisp/openwisp-utils/pull/152

### CODE REVIEW

File: openwisp_users/apps.py [hunk 1]
```diff
@@ -22,21 +23,16 @@ class OpenwispUsersConfig(AppConfig):
...
         items = [
             {'model': settings.AUTH_USER_MODEL},
             {'model': get_model_name('openwisp_users', 'Organization')},
         ]
-        if not hasattr(settings, menu_setting):
-            setattr(settings, menu_setting, items)
-        else:
-            current_menu = getattr(settings, menu_setting)
-            current_menu += items
+        register_menu_items(items)
```
Comment (nemesisdesign) [inline openwisp_users/apps.py:35]: let's abbreviate:

```python
register_menu_items([
    {'model': settings.AUTH_USER_MODEL},
    {'model': get_model_name('openwisp_users', 'Organization')},
])
```

I think the `items` variable has no reason to be kept.

File: .travis.yml [hunk 2]
```diff
@@ -30,6 +30,8 @@ script:
   - ./run-qa-checks
   - coverage run --source=openwisp_users runtests.py
   - SAMPLE_APP=1 coverage run --append --source=openwisp_users runtests.py
+  # - pip install https://github.com/openwisp/openwisp-utils/archive/master.zip
+  - pip install https://github.com/atb00ker/openwisp-utils/archive/issues/106-add-default-menu-func.zip
```
Comment (nemesisdesign) [inline .travis.yml:34]: This can now be updated to `pip install -U https://github.com/openwisp/openwisp-utils/tarball/master`

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
https://github.com/openwisp/openwisp-utils/pull/152 is merged! See my comments below.

**Comment by atb00ker** [comment]
Updated! :smile:

**Review by nemesisdesign — DISMISSED** [review]
Great, thanks you @atb00ker, will leave it open to wait until https://github.com/openwisp/openwisp-utils/issues/157 so we can increase the minimum version here once ready.

**Review by nemesisdesign — COMMENTED** [review]
@atb00ker can you upgrade the minimum openwisp-utils version to 0.7.0?

File: openwisp_users/tests/test_models.py [hunk 3]
```diff
@@ -53,6 +55,7 @@ def test_organizations_pk(self):
         self.assertIn((org1.pk,), user.organizations_pk)
         self.assertEqual(len(user.organizations_pk), 2)
 
+    @capture_stderr()
```
Comment (nemesisdesign) [inline openwisp_users/tests/test_models.py:58]: I would not hide this so that we'll remember to remove these obsolete methods in the next version

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

File: openwisp_users/tests/test_models.py (hunk 3 above)
Comment (atb00ker) [reply openwisp_users/tests/test_models.py]: Okay! 👍🏻

**Comment by atb00ker** [comment]
Okay, updated! 😄

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**